app/views/dogs/manager/_manager.html.erb

```diff
@@ -1,12 +1,25 @@
+<form class="mb-3" id="dog-search">
+  <div class="form-row">
+    <div class="col-6">
+      <input type="text" class="form-control" id='search' name='filter_params[search]' autocomplete='off' value=<%=@filter_params[:search]%> >
+    </div>
+    <div class="col-2">
+      <%= select_tag :search_field_index, options_for_select(Dog::SEARCH_FIELDS.as_options.invert, selected: @filter_params[:search_field_index] || "name"), name: "filter_params[search_field_index]", class: "form-control" %>
+    </div>
+    <div class="col-4">
+      <button type="submit" id="search_button" class="btn btn-primary">Search</button>
+    </div>
+  </div>
+</form>
```

Comment:
Not sure if we wanted to keep the `search_reset_icon` here so I didn't add it, let me know if that is preferred.